hnn_core/viz.py

```diff
@@ -270,8 +270,13 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
 
     if isinstance(dpl, Dipole):
         dpl = [dpl]
-    elif average:
-        dpl = dpl + [average_dipoles(dpl)]
+    elif isinstance(dpl, list):
+        for single_dpl in dpl:
+            _validate_type(single_dpl, Dipole, 'dpl', 'Dipole, list of Dipole')
+        if average:
+            dpl = dpl + [average_dipoles(dpl)]
+    else:
+        _validate_type(dpl, Dipole, 'dpl', 'Dipole, list of Dipole')
```

Comment:
do you even need to do this? Can't you just do:

```py
_validate_type(dpl, Dipole, 'Dipole, list of Dipole')
```

?

Reply:
I can try passing dipole and list of Dipole as types to `_validate_type()` like this 
`_validate_type(dpl,[Dipole, [Dipole]],'Dipole, list of Dipole'))`.
I will try this and let you know the results.
If there is some other way then please let me know

Reply:
I can pass `types = [Dipole,list]` only. Is there a way to pass list of Dipoles as an argument? I tried with `[Dipole]` and `list(Dipole)` but both didn't work